gravwell/dashboard/48f27640-3792-4214-beb5-77f10494372a.meta

```diff
@@ -0,0 +1,114 @@
+{
+	"UUID": "48f27640-3792-4214-beb5-77f10494372a",
+	"Name": "Gravwell Ageout Activity",
+	"Description": "Dashboard monitoring shard ageout and management activity",
+	"Data": {
+		"timeframe": {
+			"durationString": null,
+			"timeframe": "P30DT",
+			"timezone": null,
+			"start": null,
+			"end": null
+		},
+		"searches": [
+			{
+				"alias": "Shard Deletions",
+				"query": "tag=gravwell words Deleted shard\n| syslog Appname==indexer well shardts reason Message \n| regex -e Message \"Deleted (?P\u003cshardtype\u003e\\S+) shard\"\n| count by well shardtype\n| stackgraph count well shardtype",
+				"color": null
+			},
+			{
+				"alias": "Shard Deletion List",
+				"query": "tag=gravwell words Deleted shard\n| syslog Appname==indexer well shardts reason Message \n| regex -e Message \"Deleted (?P\u003cshardtype\u003e\\S+) shard\"\n| table TIMESTAMP well \"shardtype\" shardts reason",
+				"color": null
+			},
+			{
+				"alias": "Cold Migration Activity",
+				"query": "tag=gravwell words Migrated shard\n| syslog Appname==indexer well shardts reason Message~Migrated\n| regex -p -e Message \"due to hot shard (?P\u003creason\u003e.+)?\"\n| stats count by well reason over 24h\n| chart count by well reason",
+				"color": null
+			},
+			{
+				"alias": "Search 4",
```

Comment:
this appears to be a duplicate of the "Cold Migration Activity" search, maybe we can eliminate it?